src/app/helptext/system/reporting.ts

```diff
@@ -0,0 +1,20 @@
+import { T } from "app/translate-marker";
+
+export const helptext = {
+cpu_in_percentage_placeholder: T('Report CPU usage in percentage'),
```

Comment:
```suggestion
cpu_in_percentage_placeholder: T('Report CPU usage in percent'),
```

Reply:
I thought we were going to remove this option and just show CPU usage in percent only.